src/app/components/turnos/gestor-agendas/operaciones-agenda/panel-agenda.component.ts

```diff
@@ -170,8 +153,8 @@ export class PanelAgendaComponent implements OnInit {
             query['activo'] = true;
             if (this.espacioFisicoPropio) {
                 this.servicioEspacioFisico.get(query).subscribe(resultado => {
-                    if (this.agenda.espacioFisico && this.agenda.espacioFisico.id) {
-                        listaEspaciosFisicos = this.agenda.espacioFisico ? [this.agenda.espacioFisico].concat(resultado) : resultado;
+                    if (this.editaAgendaPanel.espacioFisico && this.editaAgendaPanel.espacioFisico.id) {
```

Comment:
También se podría simplificar toda la condición por: this.editaAgendaPanel.espacioFisico?.id

Reply:
todo corregido 👍🏼  gracias